doozer/doozerlib/backend/konflux_image_builder.py

```diff
@@ -0,0 +1,365 @@
+import asyncio
+import logging
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Dict, Optional, cast
+
+from artcommonlib import exectools
+from artcommonlib import util as art_util
+from dockerfile_parse import DockerfileParser
+from kubernetes import client, config, watch
+from kubernetes.client import Configuration
+from kubernetes.dynamic import DynamicClient, exceptions, resource
+from ruamel.yaml import YAML
+
+from doozerlib.backend.build_repo import BuildRepo
+from doozerlib.image import ImageMetadata
+
+yaml = YAML(typ="safe")
+LOGGER = logging.getLogger(__name__)
+
+
+class KonfluxImageBuildError(Exception):
+    def __init__(self, message: str, pipelinerun_name: str, pipelinerun: Optional[resource.ResourceInstance]) -> None:
+        super().__init__(message)
+        self.pipelinerun_name = pipelinerun_name
+        self.pipelinerun = pipelinerun
+
+
+KONFLUX_BUNDLE_PULLSPEC = "quay.io/konflux-ci/tekton-catalog/pipeline-docker-build:f3ac40bbc0230eccb8d98a4d54dabd55a4943c5d"
```

Comment:
Looks like that SHA is pretty outdated. We probably need to figure out a process, but for now we can probably use the latest build always. But I see that there is `devel` so can we try something like

`quay.io/konflux-ci/tekton-catalog/pipeline-docker-build:devel`